xilem_web/web_examples/elm/src/main.rs

```diff
@@ -0,0 +1,72 @@
+// Copyright 2024 the Xilem Authors
+// SPDX-License-Identifier: Apache-2.0
+
+use xilem_core::map_action;
+use xilem_web::{
+    document_body,
+    elements::html as el,
+    interfaces::{Element, HtmlDivElement},
+    Action, App,
+};
+
+// ----- ------
+//    Model
+// ----- ------
+
```

Comment:
I think these kinds of comments are somewhat redundant and take quite a bit of real-estate for this otherwise short example